chiselFrontend/src/main/scala/Chisel/When.scala

```diff
@@ -2,9 +2,12 @@
 
 package Chisel
 
+import scala.language.experimental.macros
+
 import internal._
 import internal.Builder.pushCommand
 import internal.firrtl._
+import internal.sourceinfo.{SourceInfo, WhenTransform}
 
 object when {  // scalastyle:ignore object.name
```

Comment:
WhenContext does not have an apply method and there are no discussions to add an apply method to WhenContext. Therefore, it does not require a macro transformation.